src/parser/core/modules/Actors/Actor.ts

```diff
@@ -59,6 +61,23 @@ class ActorResources {
 		}
 	}
 
+	private buildAttributes() {
+		const getHistoricalAttribute = (attribute: Attribute) =>
+			this.getHistoricalValue(
+				event => event.attributes?.find(eventAttr => eventAttr.attribute === attribute),
+				{attribute, value: 0, estimated: true}
```

Comment:
It'd be better if the default used the BASE_SPEED value for the tier, rather than zero.  The getEstimatedTime function doesn't safety check for "negative" time adjustments, so using a default of zero will produce somewhat nonsensical results.

Reply:
Ah yeah good catch

Reply:
Doneskies. I renamed the exported const to avoid referencing speed specifically, 'cus that value is used for all sub attrs.

Probably should be moved to another file but it can wait.